### PR TITLE
fix: CLIN-204 refactor get by ancestors endpoint

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

File: app/src/services/elastic.js [hunk 1]
```diff
@@ -668,6 +668,7 @@ export default class ElasticClient {
     async searchHPODescendants( hpo ) {
         const uri = `${this.host}/${esIndices.hpo}/_search`
         const body = {
+            size: 1000,
```
Comment (francisl) [inline app/src/services/elastic.js:671]: why hardcoded here instead of a default value as param?

Comment (adipaul1981) [reply app/src/services/elastic.js:671]: Ill add it in a constant. Its not something we will like to pass in a parameter in this endpoint. Its just to make sure that all kids a returned for a phenotype (1000 is more than enough in all cases).

**Review by francisl — APPROVED** [review]
:shipit:

**Comment by adipaul1981** [comment]
test fail (same for master and this PR)

master:
image

this PR:
image

**Pull request merged**